pmm-admin.go

```diff
@@ -422,6 +425,9 @@ When adding a MongoDB instance, you may provide --uri if the default one does no
 				os.Exit(1)
 			}
 			fmt.Println("OK, now monitoring MongoDB queries using URI", pmm.SanitizeDSN(flagMongoURI))
+			fmt.Println("It is required for correct operation that profiling of monitored MongoDB databases be enabled.")
+			fmt.Println("Note that profiling is not enabled by default because it may reduce the performance of your MongoDB server.")
+			fmt.Println("For more information read PMM documentation (https://www.percona.com/doc/percona-monitoring-and-management/conf-mongodb.html).")
```

Comment:
@AlekSi I don't like those duplicates, the whole client is like this. First thing, how to avoid those prefixes? I thought about switching from `fmt` to `log` and then use `https://godoc.org/log#SetPrefix`... or maybe you know other, better formatting library?

Reply:
Both `log` and `logrus` allow you to set prefixes.
As for duplicates – that particular text can be moved to the string constant.